Unit-01/migration/app.py

```diff
@@ -0,0 +1,106 @@
+from flask import Flask, request, redirect, url_for, render_template
```

Comment:
my feedback from your other PR applies to this file too, but don't worry about fixing things in both PRs.